packages/components/pie-divider/src/index.ts

```diff
@@ -29,8 +32,8 @@ export class PieDivider extends LitElement implements DividerProps {
                 data-test-id="pie-divider"
                 aria-hidden="true"
                 class="c-divider"
-                variant=${variant || 'default'}
-                orientation=${orientation || 'horizontal'}
+                variant=${ifDefined(variant)}
+                orientation=${ifDefined(orientation)}
```

Comment:
This is probably okay, but I think `variant` and `orientation` should always be defined because we're giving them the default values on L21 and L25